.github/scripts/install-bionic.sh

```diff
@@ -0,0 +1,35 @@
+#/bin/bash
```

Comment:
Maybe these should be their own PR to make this one smaller.

Reply:
I disagree. Getting dependencies right is part of setting up testing.